kafka-init/create_topics.py

```diff
@@ -161,13 +156,46 @@ def create_topics(default_config, existing_topics):
                 'partitions not set for topic %s, it will not be created!',
                 topic_name)
             continue
-
+        if topic_name in existing_topics:
+            logger.info('Topic %s already exists, ensuring configuration options match up',
```

Comment:
I would move this log statement into update_topic_configs(). That way if something goes wrong, we'll know which topic was being updated

Reply:
good call :)